hastexo/models.py

```diff
@@ -35,6 +35,9 @@ class Meta:
     name = models.CharField(max_length=64, db_index=True)
     student_id = models.CharField(max_length=40, db_index=True)
     course_id = models.CharField(max_length=50, db_index=True)
+    learner = models.ForeignKey(django_settings.AUTH_USER_MODEL,
```

Comment:
I'm still a bit on the fence about this being called `learner`, as opposed to calling this `user` and renaming the existing `user` field to `username`.

Reply:
As discussed elsewhere, we will stick with the `learner` field.